src/server/src/Server.cpp

```diff
@@ -270,20 +306,15 @@ void Server::set_state(const State& state)
 
 void Server::stream_robot_state()
 {
-    double position[3];
-
-    dhdGetPosition(&position[0], &position[1], &position[2]);
-
-    Vector& position_out = port_position_.prepare();
-    position_out = Vector(3, position);
-    port_position_.write();
+    double robot_state[9];
 
+    dhdGetPosition(&robot_state[0], &robot_state[1], &robot_state[2]); /* position px, py, pz */
```

Comment:
The comment should stay before the line code and not alongside it. Can you change it?